recipe/meta.yaml

```diff
@@ -10,42 +10,63 @@ source:
   sha256: bd94dd61edae3b23e43e70bbbfbada06e6d2f7fbac292ed1dbcfcb3afcca929a
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # because of restraint on pandas <2 missing for py312
+  skip: True  # [py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - setuptools
     - setuptools-scm
+    - wheel
   run:
+    - python
     - importlib-metadata
-    - python >=3.6
-    - pandas
-    - numpy >=1.15.0
+    - pandas <2
+    - numpy {{ numpy }}
     - statsmodels
     - matplotlib-base
     - pytensor
     - pymc
+    - {{ compiler('cxx') }}
```

Comment:
You might need c too no?

Reply:
why is it necessary? in case some usage of pytensor involved the compilation of pure C?

Reply:
I agree with Jean, this should be added as runtime dep for pytensor

Reply:
so should I rebuild `pytensor` and hold off on this until it's rebuilt?

Reply:
That'd be my understanding of this issue